content/blog/2020-04-parsnip-adjacent/index.Rmarkdown

```diff
@@ -0,0 +1,121 @@
+---
+title: New parsnip-adjacent packages
+slug: parsnip-adjacent
+description: >
+    Three new tidymodels packages, baguette, poissonreg, and plsmod, were just released on CRAN.
+date: 2020-04-15
```

Comment:
```suggestion
date: 2020-04-21
```

@batpigandme, @topepo, and anybody else I may need to tell here -- we would like to launch the site and publish this post next Tuesday.